src/Dfe.PlanTech.Domain/Questionnaire/Models/RecommendationChunkDbEntity.cs

```diff
@@ -21,4 +23,6 @@ public class RecommendationChunkDbEntity : ContentComponentDbEntity, IRecommenda
 
     [DontCopyValue]
     public List<RecommendationSectionDbEntity> RecommendationSections { get; set; } = [];
+
+    public CSLinkDbEntity? CSLink { get; init; }
```

Comment:
Just been testing locally and noticed that recommendation chunks are not being hooked up to this link
its because whilst we have CSLink here we don't have the corresponding Id 

You'll just need to add a CSLinkId here much like the HeaderDbEntity has a HeaderId

Reply:
ah and now it doesn't need the = null!; 😄